### PR TITLE
feat: notifications

### PR DESCRIPTION
**Pull request by meteyou**
- display moonraker announcements as a notification
- snooze & remove announcements notifications
- add klipper warnings to notifications
- add moonraker warnings to notifications
- add raspberry pi flags to notifications

Signed-off-by: Stefan Dej <meteyou@gmail.com>

### CODE REVIEW

File: src/store/server/getters.ts [hunk 1]
```diff
@@ -159,4 +159,25 @@ export const getters: GetterTree<ServerState, any> = {
...
+        /*let flags = [
+            'Under-Voltage Detected',
+            'Frequency Capped',
+            'Currently Throttled',
+            'Temperature Limit Active',
+            'Previously Under-Volted',
+            'Previously Frequency Capped',
+            'Previously Throttled',
+            'Previously Temperature Limited',
+        ]*/
```
Comment (th33xitus) [inline src/store/server/getters.ts:174]: Is this commented out code still needed?

**Review by th33xitus — CHANGES_REQUESTED** [review]

**Comment by th33xitus** [comment]
Please also refactor all other locales as much as possible so they match the en locales.

File: src/store/server/getters.ts (hunk 1 above)
Comment (meteyou) [reply src/store/server/getters.ts:174]: It's nice to have this comment for testing.

File: src/locales/de.json [hunk 2]
```diff
@@ -1,5 +1,28 @@
 {
     "App": {
+        "Notifications": {
+            "DependencyName": "Abhängigkeit: {name}",
+            "DependencyDescription": "Deine momentane {name} Version unterstützt nicht alle Funktionen von Mainsail. Aktualisiere {name} mindestens auf Version {neededVersion}.",
```
Comment (th33xitus) [inline src/locales/de.json:5]:
```suggestion
            "DependencyDescription": "Die momentane {name} Version unterstützt nicht alle Funktionen von Mainsail. Aktualisiere {name} mindestens auf Version {neededVersion}.",
```

File: src/locales/en.json [hunk 3]
```diff
@@ -1,5 +1,28 @@
 {
     "App": {
+        "Notifications": {
+            "DependencyName": "Dependency: {name}",
+            "DependencyDescription": "Your current {name} version does not support all features of Mainsail. Update {name} to at least {neededVersion}.",
```
Comment (th33xitus) [inline src/locales/en.json:5]:
```suggestion
            "DependencyDescription": "The current {name} version does not support all features of Mainsail. Update {name} to at least {neededVersion}.",
```

**Review by th33xitus — CHANGES_REQUESTED** [review]

**Review by th33xitus — DISMISSED** [review]
LGTM

**Review by th33xitus — APPROVED** [review]

**Pull request merged**